deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml

```diff
@@ -71,6 +71,11 @@ spec:
                 - force
                 - version
                 type: object
+              proceed:
+                default: true
+                description: Procceed whether to upgrade or not, this gives operator
```

Comment:
Can we add more reasoning to the description. Something like:
"Given all conditions have passed and cluster is ready to upgrade, proceed governs this decision to continue and commence the upgrade".